packages/transport/src/modules/messages/MessageController.ts

```diff
@@ -289,21 +289,21 @@ export class MessageController extends TransportController {
                 throw CoreErrors.general.recordNotFound(Relationship, recipient.toString());
             }
 
-            const signature: ICryptoSignature = await this.secrets.sign(relationship.relationshipSecretId, plaintextBuffer);
```

Comment:
It may not be a problem, but when I hover the cursor over the `signature` variable, I no longer see that it is an `ICryptoSignature` but a `CryptoSignature`.

Reply:
in this case we get an even stronger type than before